src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -120,6 +121,15 @@
         return getConfigDropinsFeatures(result, serverDirectory, bootstrapProperties, "overrides");
     }
 
+    /**
+     * Indicate that the feature names should remain unmodified. The default is to make all the names lower case.
```

Comment:
`Indicate that the feature names should remain unmodified.` -> `Indicate whether the feature names should be converted to lowercase.`

Reply:
👍